Sources/ProjectDescription/Headers.swift

```diff
@@ -42,11 +46,13 @@ public struct Headers: Codable, Equatable {
     public let exclusionRule: AutomaticExclusionRule
 
     private init(public: FileList? = nil,
+                 umbrellaHeader: Path? = nil,
                  private: FileList? = nil,
                  project: FileList? = nil,
                  exclusionRule: AutomaticExclusionRule)
     {
```

Comment:
Do we want this to be public? If we specify `public`/`private`/`project` we shouldn't be able to specify umbrella as well, right?

We probably need to keep the old init public and add this one as private

Reply:
this is private, cause has all kinds of params, old fashion init is public, but marked as deprecated